File: Python/grundlagen/zweitegrund.py
from math import *
from grundlagen import winkel

def umrechnen_koordinaten(y1,x1,y2,x2):
    t12 = atan2((y2-y1), (x2-x1))
    s12 = sqrt((y2-y1)**2 + (x2-x1)**2)
    t12 = winkel.rad2gon(t12)
    return t12, s12

File: Georechner_C++/grundlagen/zweitegrund.cpp
#include <math.h>
#include "mainwindow.h"
#include <iostream>

std::array<float, 2> umrechnen_koordinaten2(const float& y1, const float& x1, const float& y2, const float& x2){
    float t12 = atan2((y2-y1), (x2-x1));
    float s12 = sqrt(pow(y2-y1, 2) + pow(x2-x1, 2));
    //s12 = roundoff(s12, 5);
    t12 = rad2gon(t12);
    std::array<float, 2> arr_t12_s12 = {t12, s12};
    return arr_t12_s12;
}
